code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -34,6 +34,8 @@
 	icon_state = "holdingpack"
 	max_w_class = 5
 	max_combined_w_class = 35
+	var/pshoom = 'sound/items/PSHOOM.ogg'
+	var/alt_sound = 'sound/items/PSHOOM_2.ogg'

```

Comment:
These don't really need to be vars, just the path in playsound() works; unless you explicitly want both to be vareditable